### PR TITLE
feat : added home page

### PR DESCRIPTION
**Pull request by Prashantrajput11**
1. Added Homepage

- Contains Navbar
- Hero section
- Showcase Section
- Categories Section

### CODE REVIEW

File: src/components/Categories/Categories.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  // Runs only once
+  useEffect(()=>{
+    
+      // Get data from Db
+      axios.get('api/categories')
+      .then(response => setCategories(response.data.categories)) // If success update state
+      .catch(err => console.log(err)) // If error
+    },[])
+
+  // Return
```
Comment (manojasarsa) [inline src/components/Categories/Categories.js:24]: you can remove comments which are self-explantory

File: src/pages/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Info : contains data related to Home page
+
+// Imports
+import React from 'react'
+import Categories from '../../components/Categories/Categories.js'
+import {Hero} from '../../components/Hero/Hero.js'
+import Showcase from '../../components/Showcase/Showcase.js'
```
Comment (manojasarsa) [inline src/pages/Home/Home.js:7]: you can make use of barrel export here

**Review by manojasarsa — APPROVED** [review]
Rest, code looks clean.

File: src/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+.user-profile-icon .icons:hover, .user-profile-cart .icons:hover{
+  color: var(--yellow-d0);
+  font-size: 3.6rem;
+  transform: translateY(-0.3rem);
+  cursor: pointer;
+}
+
+.usericon-container{
```
Comment (Sen-442b) [inline src/components/Navbar/Navbar.css:66]: you can separate `user` and `icon` with `-`

File: src/components/Categories/Categories.js (hunk 1 above)
Comment (Prashantrajput11) [reply src/components/Categories/Categories.js:24]: Added these comments to keep a consistent comment structure. But will keep your suggestion in mind

**Review by Sen-442b — APPROVED** [review]
-  Good job with the precision of  classesNames using BEM  convention 
- UI looks great
- Theme is unique as well

**Pull request merged**